src/main/java/com/autotune/analyzer/utils/AnalyzerConstants.java

```diff
@@ -199,9 +202,10 @@ public RecommendationSubCategory[] getRecommendationSubCategories() {
 
     public enum RecommendationNotificationTypes {
         INFO("info", 1),
-        WARN("warning", 2),
-        ERROR("error", 3),
-        CRITICAL("critical", 4);
+        NOTICE("notice", 2),
+        WARN("warning", 3),
+        ERROR("error", 4),
+        CRITICAL("critical", 5);
```

Comment:
Order should be `INFO, ERROR, NOTICE, WARNING, CRITICAL`

Reply:
Updated the order. Thanks!